### DISCUSSION TITLE
About the color of the switch

### DISCUSSION THREAD

**Discussion by naskater** [violentmonkey/violentmonkey#1204] [Q&A]
Is it possible to change the color of the switch enabled and disabled by the script to more vivid and conspicuous colors?  such as green。The black doesn't look so obvious

**Comment by tophf** [comment]
You can change the entire UI of Violentmonkey using `Custom CSS` option in advanced settings.

```css
use[*|href="#toggle-on"] {
  fill: red;
}
```

Use any other [CSS color](https://www.google.com/search?q=color+picker) instead of `red`, like `#10a0e8`.

**Reply by naskater** [reply]
> You can change the entire UI of Violentmonkey using `Custom CSS` option in advanced settings.
> 
> ```css
> use[*|href="#toggle-on"] {
>   fill: red;
> }
> ```
> 
> Use any other [CSS color](https://www.google.com/search?q=color+picker) instead of `red`, like `#10a0e8`.

thank you so much！
